admin-ui/src/pages/request-replay-page.tsx

```
@@ -0,0 +1,436 @@
+    [blob?.request_body_kind, form?.bodyText],
+  )
+
+  function resetForm(): void {
+    if (!initialForm) return
+    setForm(initialForm)
+  }
+
+  /* ---- Send / streaming state ---- */
+  const [sending, setSending] = useState(false)
+  const [streaming, setStreaming] = useState(false)
+  const [collectResult, setCollectResult] = useState<ReplayCollectResult | null>(null)
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)